pytorch_pfn_extras/handler/__init__.py

```diff
@@ -0,0 +1,4 @@
+from pytorch_pfn_extras.handler._handler import BaseHandler, Handler  # NOQA
```

Comment:
Sorry I mistakenly removed the `BaseLogic, Logic` import in my previous suggestion 😅 

```suggestion
from pytorch_pfn_extras.handler._handler import BaseHandler, Handler  # NOQA
from pytorch_pfn_extras.handler._logic import BaseLogic, Logic  # NOQA
```

Reply:
sorry, it was my fault for not double-checking 😥